test/fixtures/cookbooks/acceptance/test-suite/.acceptance/acceptance-cookbook/recipes/test.rb

```diff
@@ -0,0 +1 @@
+log File.join(ENV['PWD'], __FILE__)
```

Comment:
I was thinking we can `log 'test recipe'` here, add similar `provision.rb` and `destroy.rb` files and in the output check if "test" (or "provision" or "destroy") is being outputed. We will need to add two more test cases to cover the provision and destroy. Does that make sense?